README.md

```diff
@@ -562,16 +562,12 @@ suite_tear_down
 
 ### Saving runs, Loading & comparing previous runs
 
-Benchee can store the results of previous runs in a file and then load them
-again to compare for example what was recorded against a branch with performance
-improvements.
+Benchee can store the results of previous runs in a file and then load them again to compare them. For example this is useful to compare what was recorded on the master branch against a branch with performance improvements.
 
-**Saving** is done through the `save` configuration specifying a `path`,
-where to save the results defaulting to `"benchmark.benchee"`, and a `tag`, to annotate these results (for instance with a branch name) defaulting to a time
-stamp.
+**Saving** is done through the `save` configuration specifying a `path`, where to save the results defaulting to `"benchmark.benchee"`, and a `tag`, to annotate these results (for instance with a branch name) defaulting to a time stamp.
```

Comment:
This is mainly a grammatical thing, but this sentence is a little long and reads kind of weird. How about something like this:

**Saving** is done through the `save` configuration option. You can specify a `path` where results are saved, or you can use the default option of`"benchmark.benchee"` if you don't pass a `path`. You can also pass a `tag` option which annotates these results (for instance with a branch name). The default option for the `tag` is a timestamp of when the benchmark was run.

Reply:
much better, thanks! :)